bin/glassdoor israel cities data.json

```diff
@@ -0,0 +1,1596 @@
+{
```

Comment:
That's great!`bin` directory should contain a binary files and not data files, just move it to `data` directory and remove `bin`